AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/general.py

```diff
@@ -0,0 +1,39 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+
+"""
+General constants
+"""
+BASE_LEVEL_NAME = "Base"
+SAVE_STRING = "Save"
+NAME_STRING = "Name"
+WAIT_FRAMES = 200
+WAIT_TIME_3 = 3
+WAIT_TIME_5 = 5
+EVENTS_QT = "Events"
+EVENT_NAME_QT = "EventName"
+
+VARIABLE_TYPES_DICT = {
+    "Boolean": "Boolean",
+    "Color": "Color",
```

Comment:
Why is this a dict mapping key-to-key and not a list?

Reply:
The dictionary can be iterated through when building new test code or fed into the simplenamespace class to be used like an enum but for strings. I'm going to move this over to the variable manager class since that's where it's currently used.